src/Microsoft.Health.Fhir.Core/Extensions/BulkDeleteMediatorExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Features.Operations.BulkDelete.Mediator;
+using Microsoft.Health.Fhir.Core.Messages.Delete;
+
+namespace Microsoft.Health.Fhir.Core.Extensions
```

Comment:
Love the idea of extensions per feature, should they also be kept in the feature folder?

Reply:
That makes sense. Keep all the files for the feature in one place.